corso_website_springboot/src/main/java/com/corso/springboot/utils/annotations/ValidCanadianPostalCode.java

```diff
@@ -12,7 +12,7 @@
 @Retention(RetentionPolicy.RUNTIME)
 @Constraint(validatedBy = ValidPostalCodeRegex.class)
 public @interface ValidCanadianPostalCode {
-    String message() default "Invalid Phone Number Format.";
+    String message() default "Invalid Postal Code Format.";
 
     Class<?>[] groups() default {};
```

Comment:
Can you explain the Class<?> groups() default{}; please

Reply:
It allows the specification of validation groups, to which this constraint belongs, in other words nothing important for our use case, but it's required for the Custom constraint to be a valid one.